docs/source/nodes_and_pipelines/namespaces.md

```diff
@@ -0,0 +1,345 @@
+        parameters={"params:model_options": "params:model_options_2"}, # Using a new set of parameters to train model
+        namespace = "ds_2", # With that namespace, "ds_2." prefix will be added to inputs, outputs, params, and node names
+        inputs={"model_input_table"}, # I need to list all pipeline inputs here even if I don't want to change them
+    )
+```
+
+To use a new set of parameters, copy `model_options` from `conf/base/parameters_data_science.yml` to `conf/base/parameters_data_science_2.yml` and modify it slightly to try new model training parameters, such as test size and a different feature set. Call it `model_options_2`:
+
+```python
+#conf/base/parameters.yml
+model_options_2:
+  test_size: 0.3
+  random_state: 3
+  features:
+    - d_check_complete
+    - moon_clearance_complete
+    - iata_approved
+    - company_rating
+```
+
+In this example, all nodes inside the `data_science_2` pipeline will be prefixed with `ds_2`: `ds_2.split_data`, `ds_2.train_model`, `ds_2.evaluate_model`. Parameters will be used from `model_options_2` because we overwrite `model_options` with them. The input for that pipeline will be `model_input_table` as it was previously, because we mentioned that in the inputs parameter (without that, the input would be modified to `ds_2.model_input_table`, but we don't have that table in the pipeline).
+
+Since the node names are unique now, we can run the project with:
+
+```python
+kedro run
+```
+
+Logs show that `data_science` and `data_science_2` pipelines were executed successfully with different R2 results. Now, we can see how Kedro-viz renders namespaced pipelines in collapsible "super nodes":
+
+```python
+kedro viz run
+```
+
+After running viz, we can see two equal pipelines: `data_science` and `data_science_2`:
+
+![namespaces uncollapsed](../meta/images/namespaces_uncollapsed.png)
+
+We can collapse all namespaced pipelines (in our case, it's only `data_science_2`) with a special button and see that the `data_science_2` pipeline was collapsed into one super node called `Ds 2`:
+
+![namespaces collapsed](../meta/images/namespaces_collapsed.png)
+
+> Tip: You can use `kedro run --namespace = namespace_name` to run only the specific namespace
+
+
+### How make all pipelines in this example fully namespaced
+
+If we want to make all pipelines in this example fully namespaced, we should:
+
+Modify the `data_processing` pipeline by adding to the `pipeline()` creation function in `src/project_name/pipelines/data_processing/pipeline.py` with the following code:
+```python
+        namespace="data_processing",
+        inputs={"companies", "shuttles", "reviews"},
+```
+Modify the `data_science` pipeline by adding namespace and inputs the same way as it was in `data_science_2` pipeline, but the input will be prefixed with `data_processing.`, because we added a namespace for data processing in the previous step:
```

Comment:
```suggestion
Modify the `data_science` pipeline by adding namespace and inputs in the same way as it was done in `data_science_2` pipeline, but the input will be prefixed with `data_processing.`, because we added a namespace for data processing in the previous step:
```